packages/components/modules/messages/native/ChatRoomPage/ChatRoomComponent/index.tsx

```
@@ -105,6 +105,14 @@ const ChatRoomPageComponent: FC<ChatRoomPageComponentProps> = ({ roomId }) => {
     console.log('Not implemented yet.')
   }
 
+  const handleDeleteChat = () => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'handleDeleteChat'. Consider to delete it or rename it.

'handleDeleteChat' is defined here:


(lint/suspicious/noRedeclare)